Tools/scripts/deepfreeze.py

```diff
@@ -170,6 +171,8 @@ def generate_bytes(self, name: str, b: bytes) -> str:
     def generate_unicode(self, name: str, s: str) -> str:
         if s in identifiers:
             return f"&_Py_ID({s})"
+        if all(i in ascii_letters or i == "_" or i.isdigit() for i in s):
```

Comment:
A regex would look better here.
```suggestion
        if re.match(r"\A\w+\Z", s):
```

Reply:
Done